docs/components/DocumentationSidebarRightLink.tsx

```diff
@@ -140,13 +150,15 @@ const DocumentationSidebarRightLink = React.forwardRef<HTMLAnchorElement, Sideba
           style={{ paddingLeft }}
           href={'#' + slug}
           onClick={onClick}
-          css={[
-            STYLES_LINK,
-            isNested && STYLES_LINK_HEADER,
-            isCode && STYLES_LINK_CODE,
-            isActive && STYLES_LINK_ACTIVE,
-          ]}>
-          {displayTitle}
+          css={[STYLES_LINK, isNested && STYLES_LINK_HEADER, isActive && STYLES_LINK_ACTIVE]}>
+          <span css={[STYLES_LINK_LABEL, isCode && STYLES_LINK_CODE]}>{displayTitle}</span>
+          {tags && tags.length ? (
+            <div css={css({ display: 'inline-flex' })}>
+              {tags.map((tag: string) => (
+                <Tag toc name={tag} />
+              ))}
+            </div>
+          ) : undefined}
```

Comment:
```suggestion
          ) : null}
```
I always thought returning `null` was a better practice, but I can't find any proof for that after some google searching other than some developers saying it in some old chats (but there's no source/reason). So, take this suggestion if you want.

Reply:
I prefer using `undefined` in TS React, since it produces cleaner typing if you pass elements as props. 

Example:
`xxx?: React.ReactNode` vs `xxx?: React.ReactNode | null`

Also, with React 18, the biggest `undefined` disadvantage is removed:
* https://github.com/reactwg/react-18/discussions/75

A nice, in-depth article - https://medium.com/@o_o_o/null-vs-undefined-can-i-use-only-one-a3b7db5468f2

Reply:
Generally I agree with @jonsamp, in my opinion `undefined` shouldn't be used explicitly in the code. Maybe in React it doesn't matter that much, but in JS world the meaning of `undefined` is that it's an unexpected value. If something returns `null` instead, you know that it was intentional.

Reply:
I agree that `null` feels intentional. However, inside React components, it seems like we can also use `undefined` (after reading this article and considering it some). I am fine with keeping `undefined` if we can merge this. Later, if someone has a strong opinion, we can refactor these + many other spots to use `null`, but I don't think now is the right time to make those changes/patterns.

This is a case where I think everyone can read this without issue. There's no technical reason to change it. So it's a style preference. Also, it doesn't add tech debt (well, any more than all code is tech debt eventually).